esignet-insurance.properties

```diff
@@ -484,7 +484,67 @@ mosip.esignet.vci.key-values={\
       }} \
     }},\
     'display': {{'name': 'Insurance', 'locale': 'en'}}\
-  } \
+  }, \
+     'latest' : {\
+              'credential_issuer': '${mosip.esignet.vci.identifier}',   \
+              'credential_endpoint': '${mosip.esignet.domain.url}${server.servlet.path}/vci/credential', \
+              'display': {{'name': 'Insurance', 'locale': 'en'}},\
+              'credentials_supported' : { \
+                 "InsuranceCredential" : {\
+                    'format': 'ldp_vc',\
+                    'scope' : 'sunbird_rc_insurance_vc_ldp',\
+                    'cryptographic_binding_methods_supported': {'did:jwk'},\
+                    'cryptographic_suites_supported': {'Ed25519Signature2020'},\
+                    'proof_types_supported': {'jwt'},\
+                    'credential_definition': {\
+                      'type': {'VerifiableCredential','InsuranceCredential'},\
+                      'credentialSubject': {\
+                            'fullName': {'display': {{'name': 'Name','locale': 'en'}}}, \
+                            'mobile': {'display': {{'name': 'Phone Number','locale': 'en'}}},\
+                            'dob': {'display': {{'name': 'Date of Birth','locale': 'en'}}},\
+                            'gender': {'display': {{'name': 'Gender','locale': 'en'}}},\
+                            'benefits': {'display': {{'name': 'Benefits','locale': 'en'}}},\
+                            'email': {'display': {{'name': 'Email Id','locale': 'en'}}},\
+                            'policyIssuedOn': {'display': {{'name': 'Policy Issued On','locale': 'en'}}},\
+                            'policyExpiresOn': {'display': {{'name': 'Policy Expires On','locale': 'en'}}},\
+                            'policyName': {'display': {{'name': 'Policy Name','locale': 'en'}}},\
+                            'policyNumber': {'display': {{'name': 'Policy Number','locale': 'en'}}}\
+                     }},\
+                    'display': {{'name': 'Sunbird Insurance', \
+                                  'locale': 'en', \
+                                  'logo': {'url': 'https://sunbird.org/images/sunbird-logo-new.png','alt_text': 'a square logo of a Sunbird'},\
+                                  'background_color': '#FDFAF9',\
+                                  'text_color': '#7C4616'}},\
+                    'order' : {'fullName','policyName','policyExpiresOn','policyIssuedOn','policyNumber','mobile','dob','gender','benefits','email'}\
+                 },\
+                  "LifeInsuranceCredential":{\
```

Comment:
In the collab we have only the insurance Credentail, we should remove this section

Reply:
resolve in this [055a750](https://github.com/mosip/mosip-config/pull/6118/commits/055a75007f8c775f9c228207bae59af405af66df)